lib/matplotlib/axes/_axes.py

```diff
@@ -5504,9 +5504,10 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
             which can be set by *filterrad*. Additionally, the antigrain image
             resize filter is controlled by the parameter *filternorm*.
 
-        alpha : scalar, optional
+        alpha : scalar or array_like, optional, default: None
```

Comment:
```suggestion
        alpha : scalar or array-like, optional
```
I wouldn't mention the default here, as it's just a placeholder for "do not use alpha" (and even that is not unambiguously clear because None can also mean "use some default").